compiler+runtime/src/cpp/jank/runtime/core/seq.cpp

```diff
@@ -1178,4 +1179,29 @@ namespace jank::runtime
   {
     return make_box<obj::repeat>(n, val);
   }
+
+  object_ptr sort(object_ptr const coll)
+  {
+    return visit_seqable(
+      [](auto const typed_coll) -> object_ptr {
+        native_vector<object_ptr> vec;
+        for(auto it(typed_coll->fresh_seq()); it != nullptr; it = it->next_in_place())
+        {
+          vec.push_back(it->first());
+        }
+
+        std::stable_sort(vec.begin(), vec.end(), [](object_ptr const a, object_ptr const b) {
+          return runtime::compare(a, b) < 0;
+        });
+
+        using T = typename decltype(typed_coll)::value_type;
+
+        if constexpr(behavior::metadatable<T>) {
+          return make_box<obj::native_vector_sequence>(typed_coll->meta, std::move(vec));
+        } else {
+          return make_box<obj::native_vector_sequence>(std::move(vec));
```

Comment:
Please be sure to format all of these files with `clang-format`. We have a `bin/format` script to help with that, but having your editor do it for you will be best.